### PR TITLE
Richard L. Rogers- Social Media Inheritance Project

### PR DESCRIPTION
**Pull request by rlrogers**
S.O. to Jordan!

### CODE REVIEW

**Comment by santiagobasulto** [comment]
This PR will be reviewed by @eggarcia28.

File: social_network/posts.py [hunk 1]
```diff
@@ -1,33 +1,48 @@
...
+        self.timestamp = timestamp
+
+    def set_user(self, user):
+        self.user = user
+    
+    def __repr__(self):
+        rep = "{}'s post @{}".format(self.user, self.timestamp.strftime('%A, %b %d, %Y'))
+        return rep
+        
+class TextPost(Post):  # Inherit properly
```
Comment (eggarcia28) [inline social_network/posts.py]: I know it's not really that much extra code, but it seemed like a good time to practice using super().

File: social_network/accounts.py [hunk 2]
```diff
@@ -1,13 +1,22 @@
...
+        
 
     def add_post(self, post):
-        pass
+         self.posts.append(post)
 
     def get_timeline(self):
-        pass
-
+        posts = []
```
Comment (eggarcia28) [inline social_network/accounts.py]: This was cleaner than the way I did it. Good job!

File: social_network/posts.py (hunk 1 above)
Comment (eggarcia28) [reply social_network/posts.py]: I didn't really mark the code I reviewed properly, but I hope you understand what I was trying to say.

**Comment by santiagobasulto** [comment]
Thanks for reviewing @eggarcia28!

**Pull request closed without merging**